adaptive_scheduler/monitoring/seeing.py

```diff
@@ -0,0 +1,68 @@
+import abc
+
+from adaptive_scheduler.monitoring.opensearch_seeing import get_seeing
+
+
+
+class SeeingMonitor(object):
+    '''Abstract class for updating and retrieving current seeing value on a resource. Resources are meant to be updated once at the
+       beginning of each scheduling run, and the retrieved as needed throughout that run. The structure of seeing data should look like this:
+
+       seeing_by_resource = {
+         'telescope.enclosure.site': {
+            'time': datetime object,
+            'seeing': value in arcseconds
+         }
+       }
+    '''
+
+    __metaclass__ = abc.ABCMeta
```

Comment:
What's the purpose of defining the metaclass? Just curious because I genuinely haven't done this before

Reply:
I think this is what setting the class to an ABC does internally, so I removed this when making that change.